CONTRIBUTING.md

```diff
@@ -64,12 +64,14 @@ Once Homebrew is installed, open a **Terminal** window and run the following one
 
 ### Linux
 
-You should install Git, Cargo, Rust and LLVM using your distro's package manager. OpenGL, GLU and ALSA development libraries are also required.
+You should install Git, Cargo, Rust, GCC and LLVM using your distro's package manager. OpenGL, GLU and ALSA development libraries are also required.
```

Comment:
Why do we now need GCC as well as LLVM? Seems a bit steep.

Reply:
I think it was required for some crates dependent on C code. At least after reinstallation I had to install it.